lib/screens/main_screen.dart

```diff
@@ -82,10 +83,12 @@ class _MainScreenState extends State<MainScreen> {
             log('Polling is enabled.', name: 'main_screen.dart');
             _pollTimer = Timer.periodic(
                 Duration(seconds: 10), (_) => _pollForRequests());
+            setState(() => _allowManualRefresh = true);
```

Comment:
i think the gesture refresh should be independent from the timer refresh configuration

Reply:
Yes, I will link it to the existence of push tokens instead.